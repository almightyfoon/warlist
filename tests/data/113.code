ca501b_-0tcVaPaPaPaQbvbfaObubbbeb9b9___-0tgNaPaPbvcOeJeJeJaObbbbbebecob9b9