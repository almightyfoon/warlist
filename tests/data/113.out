https://conflictchamber.com/#ca501b_-0tcVaPaPaPkHbvbfaObubbbeb9b9___-0tgNaPaPbvcOeJeJeJaObbbbbebecob9b9

Legion Army - 78 / 75 points
[Theme] Oracles of Annihilation

!!! This army must contain 75 points or fewer

[Lylyth 3] Lylyth, Reckoning of Everblight [+29]
 - Angelius [17]
 - Angelius [17]
 - Angelius [17]
 - Carnivean [18]
 - Naga Nightlurker [8]
 - Nephilim Bolt Thrower [11]
 - Seraph [14]
 - Succubus [0(4)]
Blighted Nyss Shepherd [1]
Blighted Nyss Sorceress & Hellion [0(6)]
The Forsaken [0(4)]
The Forsaken [4]

Legion Army - 79 / 75 points
[Theme] Oracles of Annihilation

!!! This army must contain 75 points or fewer

[Fyanna 2] Fyanna, Torment of Everblight [+28]
 - Angelius [17]
 - Angelius [17]
 - Naga Nightlurker [8]
 - Nephilim Bloodseer [8]
 - Neraph [12]
 - Neraph [12]
 - Neraph [12]
 - Seraph [14]
Blighted Nyss Shepherd [1]
Blighted Nyss Shepherd [1]
Blighted Nyss Sorceress & Hellion [0(6)]
Blighted Nyss Sorceress & Hellion [0(6)]
Spell Martyr [1]
The Forsaken [0(4)]
The Forsaken [4]
